### ISSUE TITLE
False positive ANSIBLE0006 when doing yum clean

### ISSUE THREAD

**Issue by GaboFDC** [willthames/ansible-lint#369]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.6.4
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/Users/gabo/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.4 (default, Mar  6 2018, 18:51:36) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]


ansible-lint --version
ansible-lint 3.4.23
```

# Desired Behaviour

I'm not totally sure if this is a bug, or a request, but here is what's going on:

yum module doesn't allow to make other yum commands like `yum clean packages`. They recommend doing command for this.

Ansible lint should be able to recognize this and let you do it.

# Actual Behaviour (Bug report only)
If yo do it with commad like:

```yaml
- name: Cleaning rpm packages
  command: yum clean packages
```

Ansible-lint will complain and say:
`ANSIBLE0006 yum used in place of yum module`

But there is no actual way to solve this, as you can't do it by the yum module.

**Comment by willthames** [comment]
Sure, it's a best guess. In this situation I recommend using `warn=no` with the `command` module.
